### DISCUSSION TITLE
Contributors

### DISCUSSION THREAD

**Discussion by adamjstewart** [torchgeo/torchgeo#1906] [General]
Hello!

TorchGeo is applying to join the [OSGeo incubation program](https://www.osgeo.org/about/committees/incubation/). One of the questions they ask is:

> Please provide the names, emails and entity affiliation of all official committers.

We have figured this out for most contributors, except the following people:

* @remtav
* @ntw-au 
* @eltociear 
* @Niro4 
* @DimitrisMantas

If you are comfortable sharing some or all of this information with me and with OSGeo, please message me on one of the following channels:

* Slack (there is a button to join on our README)
* Email (my email is on my GitHub profile)
* Reply to this GitHub discussion

For anyone else who I didn't list, if you're worried that your information on GitHub is outdated, want to know which information I found, or don't want me to disclose this information to OSGeo, feel free to ask via one of the above channels.

P.S. Thanks again for your valuable contributions to TorchGeo!

**Comment by eltociear** [comment]
@adamjstewart 
No problem!

**Comment by DimitrisMantas** [comment]
Name: Dimitris Mantas
Email: dimitris.mantas@outlook.com
Affiliation: Delft University of Technology

My contributions to Torchgeo are my copyright, not my uni's, so you can also leave the affiliation field blank for me if possible.
